data/db-schema/src/main/resources/net/corda/db/schema/config/migration/cpx-creation-v1.0.xml

```diff
@@ -56,9 +56,6 @@
                              schemaName="${schema.name}"/>
 
         <createTable tableName="cpk_metadata" schemaName="${schema.name}">
```

Comment:
Table definition:
```
create table cpk_metadata
(
    cpk_name                varchar(255)            not null,
    cpk_version             varchar(255)            not null,
    cpk_signer_summary_hash varchar(255)            not null,
    file_checksum           varchar(255)            not null
        constraint db_cpk_metadata_uc1
            unique,
    format_version          varchar(12)             not null,
    metadata                text                    not null,
    entity_version          integer                 not null,
    is_deleted              boolean                 not null,
    insert_ts               timestamp default now() not null,
    constraint cpk_metadata_pk
        primary key (cpk_name, cpk_version, cpk_signer_summary_hash)
);
```

Reply:
Should `cpk_signer_summary_hash` be nullable?

Reply:
No. It shouldn't be null. 
We don't currently check it, but once the packaging changes go in, it'll always be populated.